### PR TITLE
Add issue templates like in istio/istio

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

**Comment by ssuchter** [comment]
How about directing the user to discuss.istio.io as an option (for best effort support?)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Report a bug to help us improve Istio
+---
+
```
Comment (mandarjog) [inline .github/ISSUE_TEMPLATE/bug_report.md]: We should point users to the `discuss forum` right here for "how do I do this" type questions

**Comment by geeknoid** [comment]
@ssuchter @mandarjog Added a note about discuss.istio.io

**Comment by ozevren** [comment]
/lgtm

Should we even allow bugs on the istio/api repo?

**Pull request merged**